flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/Climb.flt

```diff
@@ -153,6 +151,43 @@ A32NX_SEC_2_PUSHBUTTON_PRESSED=1
 A32NX_SEC_3_PUSHBUTTON_PRESSED=1
 A32NX_FAC_1_PUSHBUTTON_PRESSED=1
 A32NX_FAC_2_PUSHBUTTON_PRESSED=1
+A32NX_RMP_L_TOGGLE_SWITCH = 1
+A32NX_RMP_R_TOGGLE_SWITCH = 1
+A32NX_RMP_L_SELECTED_MODE = 1
+A32NX_RMP_R_SELECTED_MODE = 2
+A32NX_RMP_L_VHF2_STANDBY_FREQUENCY = 124275000
+A32NX_RMP_L_VHF3_STANDBY_FREQUENCY = 135475000
+A32NX_RMP_R_VHF1_STANDBY_FREQUENCY = 129675000
+A32NX_RMP_R_VHF3_STANDBY_FREQUENCY = 136975000
+A32NX_RMP_L_SAVED_COURSE_VOR = 22
+A32NX_RMP_L_SAVED_COURSE_ILS = 202
+A32NX_RMP_R_SAVED_COURSE_VOR = 87
+A32NX_RMP_R_SAVED_COURSE_ILS = 271
```

Comment:
Test values left behind?

Reply:
> Test values left behind?

No, same lines as other .flt files.

What's the problem?

Reply:
Does the aircraft power up with these values?

Reply:
> Does the aircraft power up with these values?

Yes, these are the default values

Reply:
The plane reads these files at startup right?

Reply:
I mean on the real plane. Our initial values should match the real aircraft power-up state wherever possible.